### PR TITLE
Expose less error formatting function

### PR DESCRIPTION
**Pull request by btd**
It looks like:

### CODE REVIEW

**Comment by contra** [comment]
I think it would be smarter if we ripped the guts out of the less error logger (error -> display portion) so our users don't need to interface with logError at all

**Comment by btd** [comment]
I did it similar way like `gulp-sass` expose own `logError` function. In this case user can choose how to process error if he need it (like gulp-plumber and friends tools).
Am i understand right you suggest to to do not write output in logError, but use function to just format error (it should be `less.engine.formatError` iirc)?

**Comment by contra** [comment]
@btd Streams are supposed to emit proper errors - we should be emitting an error that has been formatted already, not tell our users to run our errors through a different formatting function. gulp-sass is doing it wrong.

**Comment by btd** [comment]
Sorry, i forgot to update PR. I added one more filed to error with formatted message.

File: index.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ module.exports = function (options) {
     }).then(function(file) {
       cb(null, file);
     }).catch(function(err) {
-      // Convert the keys so PluginError can read them
+      // Add the keys so PluginError can read them
+      err.formattedMessage = less.engine.formatError(err, { color: true });
```
Comment (contra) [inline index.js]: why "formattedMessage"? This won't be displayed anywhere - this should be `err.message` if you want it to show up when logged

**Pull request closed without merging**